neofetch

```diff
@@ -2772,6 +2781,14 @@ dynamicprompt() {
     # Get cursor position
     info_height="$(IFS=';' builtin read -srdR -t 1 -d c -p $'\033[6n\033[c' ROW COL; printf "%s" "${ROW#*[}")"
 
+    # On Windows 10 subsystem for Linux, the default terminal does not
+    # produce any integer output, so a workaround is used.
+    if [[ $info_height =~ ^-?[0-9]+$ ]]; then
```

Comment:
Personally, I don't like this approach. A better solution like `[ -z $info_height ]` is considered but Windows' output of `$info_height` is not empty either so it can't be done. So for now I'll just use the one that works.